lib/model/QubitInformationObject.php

```diff
@@ -2286,10 +2286,13 @@ public function setHistoryByOrigination($history)
 
   public function setLevelOfDescriptionByName($name)
   {
-    // don't proceed with empty $name, or if 'otherlevel' value is passed
+    // Normalize name of "Record group" level of description (for EAD imports)
+    $name = ($name == 'recordgrp') ? 'Record group' : $name;
```

Comment:
What about other cultures? For example, if `recordgrp` is used in a French import.